api/AnalyserNode.json

```diff
@@ -77,10 +77,10 @@
               "version_added": "42"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": false
```

Comment:
The constructor was just recently implemented: https://trac.webkit.org/changeset/265196/webkit